api-extractor/src/ResolvedApiItem.ts

```diff
@@ -75,12 +79,16 @@ export default class ResolvedApiItem {
     deprecatedMessage: IDocElement[],
     isBeta: boolean,
     params:  {[name: string]: IParam},
-    returnsMessage: IDocElement[]) {
+    returnsMessage: IDocElement[],
+    apiTag: ApiTag,
+    apiItem?: ApiItem) {
```

Comment:
@pgonzal What are your thoughts on this optional parameter? This allows us to have a handle on the ApiItem to call 'ApiItem.tryCompletingReferences()' when we are in stage 2 for ApiItem initialization (i.e. when we obtain a ResolvedApiItem instance in 'DocElementParser.parseInheritDoc()'